src/Microsoft.DotNet.Interactive.Formatting/PlainTextFormatter{T}.cs

```diff
@@ -107,10 +107,14 @@ internal static PlainTextFormatter<T> CreateForAnyEnumerable()
 
         return new((value, context) =>
         {
+            using var a = context.IncrementDepth();
+            using var b = context.IncrementDepth();
```

Comment:
The variables (`a` and `b`) seem to be unused - I am guessing you added them for testing and intended to remove them before merge.

Reply:
They are being used because we're `using` them. 😂 (Sorry, that pun was right there.) In other words, `Dispose` is being called on them.

The variable names are awful.

Reply:
I often use `_` as a variable name for these situations. I suppose `_` and `__` can work.

Reply:
Ah I missed the `using` keyword 😄